.github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml

```diff
@@ -0,0 +1,38 @@
+name: "Feature Request"
+description: Create a new ticket for a new feature request
+title: "[REQUEST]: "
+labels: ["enhancement", "triage"]
+
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: "Summary"
+      description: Provide a brief explanation of the feature
```

Comment:
A brief summary is fine, but it doesn't look like there's anywhere for a more detailed explanation. I might be missing something though.